integration-tests/configure_product.py

```diff
@@ -191,7 +191,6 @@ def modify_datasources():
                         database_names.append(database_name)
             artifact_tree.write(file_path)
 
-
```

Comment:
Because of following rules, Shall we add this line back and use these rules for other methods as well. :) 

1. Two blank lines between the import statements and other code.
2. Two blank lines between each function.

You can find further details friom [here](https://www.python.org/dev/peps/pep-0008/).

Reply:
Fixed. Please refer latest commit.